src/win/filepath.c

```diff
@@ -0,0 +1,62 @@
+/*
+    This file is part of darktable,
+    copyright (c) 2009--2012 johannes hanika.
+    copyright (c) 2010--2012 tobias ellinghaus.
```

Comment:
Please remove those lines. You can probably add your name here but I feel it should better to remove all copyright in the sources. Does not makes sense at this time almost no one update them.